chainntnfs/txnotifier.go

```diff
@@ -840,7 +840,7 @@ func (n *TxNotifier) RegisterSpend(ntfn *SpendNtfn) (*HistoricalSpendDispatch,
 	if err == nil {
 		if hint > startHeight {
 			Log.Debugf("Using height hint %d retrieved from cache "+
-				"for %v", startHeight, ntfn.SpendRequest)
+				"for %v", hint, ntfn.SpendRequest)
```

Comment:
Would still be useful to log the OG height I think.

Reply:
Fixed.